lib/include/fastdds_qos_profiles_manager/data_reader/qos/Qos.hpp

```diff
@@ -0,0 +1,104 @@
+// Copyright 2023 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @file
+ */
+
+#ifndef _FAST_DDS_QOS_PROFILES_MANAGER_DATA_READER_QOS_QOS_HPP_
```

Comment:
I do not think the folder `qos` is needed. There is going to be only one file within, is not it? If so, I think it is better to leave the file in the root folder. In DomainParticipant the first level API was introduced in the same folder as `DomainParticipant.hpp`.